### DISCUSSION TITLE
CustomCellのボタンをタップしたい

### DISCUSSION THREAD

**Discussion by marinamikr** [lit-iphone-course/lit-iphone-course#32] [Question-original] [answered]
深夜に申し訳ありません。
(1) 実現したいこと
 CustomCellのGoというボタンをタップした時
number = number + 1という作業をしたい。

(2) 実現するために自分が試した内容とその結果
下の記事を参考にして実装しましたがボタンをタップしても何も起こりませんでした。そもそもCustomCellDelegateが呼ばれていないような気がします。

(3) 調べた内容や資料
https://qiita.com/HikaruKuroda/items/024d9da4369602e6cf9d

(4) 原因だと思われそうな箇所(ソースコードをコピペして貼る)
https://github.com/marinamikr/LA-Original/blob/main/ExamFriend/ViewController/TimeLineViewController.swift#L144-L149
https://github.com/marinamikr/LA-Original/blob/main/ExamFriend/TimeLineTableViewCell.swift#L37-L39

(5) GitHubのリンク
https://github.com/marinamikr/LA-Original

**Comment by txxxxc** [comment, marked as answer]
CustomCellのdelegateにTimeLineViewControllerを設定できていないからだと思います！なので、cellの初期化時に
`cell.delegate = self`みたいな感じで書くとdelegateメソッドが呼ばれるはずなので、試してみてくださいー！

**Reply by marinamikr** [reply]
完全に見逃していました🙇🏻‍♀️🙇🏻‍♀️
上手くいきました！
ありがとうございます！！！
